packages/synapse-react-client/src/components/Forum/DiscussionReply.tsx

```diff
@@ -46,8 +48,22 @@ export function DiscussionReply(props: DiscussionReplyProps) {
 
   const isCurrentUserAuthor = reply.createdBy == currentUserProfile?.ownerId
 
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search)
+    const replyId = params.get('replyid')
+    if (replyId === reply.id && replyRef.current) {
+      replyRef.current.style.transition = 'background-color 1s ease'
+      replyRef.current.style.backgroundColor = '#fbf4e0'
+      setTimeout(() => {
+        if (replyRef.current) {
+          replyRef.current.style.backgroundColor = 'transparent'
+        }
+      }, 2000)
+    }
```

Comment:
Cleanup side effect in case this component unmounts before `setTimeout` runs
```suggestion
    let timerId
    if (replyId === reply.id && replyRef.current) {
      replyRef.current.style.transition = 'background-color 1s ease'
      replyRef.current.style.backgroundColor = '#fbf4e0'
      timerId = setTimeout(() => {
        if (replyRef.current) {
          replyRef.current.style.backgroundColor = 'transparent'
        }
      }, 2000)
    }
    return () => {
      if (timerId) {
        clearTimeout(timerId)
      }
    }
```